dracut/30ignition/module-setup.sh

```diff
@@ -77,6 +77,7 @@ install() {
 
     # units only started when we have a boot disk
     # path generated by systemd-escape --path /dev/disk/by-label/root
+    install_ignition_unit ignition-diskful-udev.service    ignition-diskful.target
```

Comment:
Ah but we are starting this target by default then?  I think it's unfortunate to introduce a `systemd-udev-settle.service` dependency in the default boot - that service is considered deprecated because the systemd philosophy is that there isn't a specific "settled" point, devices can come and go.

So this goes back to "make this conditional on a kernel argument" or so?

Reply:
Revised. See the latest.